testsuite/features/secondary/proxy_container_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2021-2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# We also test 'Bootstrapping using the command line' in this feature with the following script:
+# https://github.com/uyuni-project/uyuni/blob/master/java/conf/cobbler/snippets/minion_script
```

Comment:
Is this still valid?

Reply:
I don't know much about this thing, but it looks like it is still being used:
```
testsuite/features/secondary/proxy_container_cobbler_pxeboot.feature:    And I attach the file "/sle-15-sp4-autoyast.xml" to "fileUpload"
```
and
```
testsuite/features/upload_files/sle-15-sp4-autoyast.xml:          $SNIPPET('spacewalk/minion_script')
```